src/cmd-generate-hashlist

```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/env python3
+"""
+Creates a KeyLime Hashlist for an image.
+
+See: https://keylime.dev/
+"""
+
+import argparse
+import datetime
+import json
+import os
+import subprocess
+import sys
+import tempfile
+
+cosa_dir = os.path.dirname(os.path.abspath(__file__))
+sys.path.insert(0, cosa_dir)
+
+from cosalib import meta
+from cosalib.cmdlib import (
+    sha256sum_file,
+    import_ostree_commit)
+
+
+class HashListV1(dict):
+    """
+    Abstraction of a HashList in the version 1 series
```

Comment:
Link to the JSON spec somewhere around here would be helpful.

Reply:
Getting an official document of the JSON format into the keylime documentation is something I'm working on in the next couple of days. Then there will be something to link to